lambdalib/iolib/README.md

```diff
@@ -0,0 +1,80 @@
+# IOLIB
+
+## Cell Listing
+
+| Cell                             | Type    | Description                 |
+| ---------------------------------|---------|-----------------------------|
+[[la_iobidir](./rtl/la_iobidir.v)  | Digital | Bidirectional
+[la_ioinput](./rtl/la_ioinput.v)   | Digital | Input
+[la_ioxtal](./rtl/la_ioxtal.v)     | Digital | Xtal tranceiver
+[la_iorxdiff](./rtl/la_iorxdiff.v) | Digital | Differential input
+[la_iotxdiff](./rtl/la_iotxdiff.v) | Digital | Differential output
+[la_ioanalog](./rtl/la_ioanalog.v) | Analog  | Pass through ESD protection
+[la_iovdd](./rtl/la_iovdd.v)       | Supply  | Core power
+[la_iovss](./rtl/la_iovss.v)       | Supply  | Core ground
+[la_iovddio](./rtl/la_iovddio.v)   | Supply  | IO  power
+[la_iovssio](./rtl/la_iovssio.v)   | Supply  | IO ground
+[la_iovdda](./rtl/la_iovdda.v)     | Supply  | Analog power
+[la_iovssa](./rtl/la_iovssa.v)     | Supply  | Analog ground
+[la_iopoc](./rtl/la_iopoc.v)       | Supply  | Power on control
+[la_iocorner](./rtl/la_iocorner.v) | Supply  | Corner connector
+[la_ioclamp](./rtl/la_ioclamp.v)   | Supply  | ESD clamp
+[la_iocut](./rtl/la_iocut.v)       | Supply  | Power ring cutter
+
+## PARAMETERS
+
+### CFGW
+The `CFGW` parameter defines the width of the configuration bus of the io cell. IO cells generally include a set of configuration inputs for things like drive strength and operating modes. Setting `CFGW` to a large value (eg. 16/32) should have zero impact on the design as the extra bus bits get optimized away during implementation.
+
+For la_bidir, the first 8 bits of the configuration bus are reserved for the functionality shown in the table below.
+
+| Bit       | Description                                     |
+|-----------|-------------------------------------------------|
+ CFG[0]     | slew rate control (0=fast, 1 =slow)             |
+ CFG[1]     | schmitt trigger select (0=CMOS, 1=schmitt)      |
+ CFG[2]     | pull enable (0=no pull, 1=enables weak pull)    |
+ CFG[3]     | pull select (1=pull up, 0=pull down)            |
+ CFG[7:4]   | drive strength                                  |
+
+#### RINGW
+The `RINGW` parameter specifies the number of signals within the power bus that connects all of the io cells together within the padring.
```

Comment:
I think we should indicate that these signals should be set by the tech specific lambdalib implementation via a `include "la_padring_cfg.vh"`or something like that. This could be set inside the la_padring.